src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/AssociatedCaseLinkHelperTest.java

```diff
@@ -129,6 +180,34 @@ public void shouldLinkCaseByNinoButNotToOthersIfCaseIdIsNull() {
 
     @Test
     public void givenMultipleAssociatedCases_thenAddAllAssociatedCaseLinksToCase() {
+        associatedCaseLinkHelper = new AssociatedCaseLinkHelper(ccdService, idamService, updateCcdCaseService, false);
```

Comment:
you can use ReflectionTestUitls to set value to false rather than recreating the object ?